docs/community/maintainers/release-process.md

```diff
@@ -47,24 +47,15 @@ For example, the first Nebari CalVer release was `2022.10.1`. If a hotfix releas
 
 We use the following guidelines to manage `git` branches by assigning certain roles to particular branches.
 
-- [`develop`](https://github.com/nebari-dev/nebari/tree/develop) - Represents the active development branch and is the _default_ branch on the GitHub repository.
+- [`main`](https://github.com/nebari-dev/nebari/tree/main) - Represents the active development branch and is the _default_ branch on the GitHub repository.
 
-- [`main`](https://github.com/nebari-dev/nebari/tree/main) - Represents a production-ready state of the code-base, with an appropriate tag to match the most recent release.
+## Release Tags
 
-- `release/YYYY-MM-releaseNumber` - Represents the branch for the upcoming release and only briefly exist while actively preparing for the release.
+- `YYYY-MM-releaseNumber` - Represents the tag for a particular release.
 
 ### Process
 
-Although this process is captured in the [release checklist template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=release-checklist.md&title=%5BRELEASE%5D+%3Cversion%3E), it's worth making clear how branches are managed.
-
-- Active development occurs against the `develop` branch.
-- When it's time for a release, the Release Captain will create the release branch `release/YYYY-MM-releaseNumber` and prepare the branch for the release. At times, this might mean cherry-picking commits that are needed for this release and at other times, this might mean merging `develop` into this release branch.
-- As soon as this release branch is ready, the Release Captain can open a pull request against `main`. From here, all of the changes that are included in the release should be visible in the "Files changed" section of the pull request.
-- Once CI passes, all manual tests are successful and the team is happy with the changes, the Release Captain can complete the release checklist and cut the release.
-
-#### Hotfixes
-
-In the event that a patch or hotfix release is needed, release process is the same as outlined above. The only difference is that the commits that are merged into the hotfix release branch will need to be cherry-picked from the `develop` branch.
+The release process is captured in the [release checklist template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=release-checklist.md&title=%5BRELEASE%5D+%3Cversion%3E).
```

Comment:
I think this is good to keep:
```
#### Hotfixes

If a patch or hotfix release is needed, the release process is the same as outlined above. The only difference is that the commits merged into the hotfix release branch will need to be cherry-picked from the `main` branch.
```

Reply:
Yes, at the time I thought we didn't want to do hotfixes this way anymore, but we promptly did exactly that.  Happy to add it back in.

Reply:
I looked at this again today.  The hotfix release process is also captured in the template that is linked to so I don't see the need for an explanation here as well.  I think it's easier to keep the docs in sync if we just point to one place for updates rather than needing to update various places.